underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -219,8 +220,9 @@ public FileSystem load(String userKey) throws Exception {
       }
     } catch (Exception e) {
```

Comment:
Print the exception as well, in case it's not a version problem (like class not found)?